idd/Energy+.idd.in

```diff
@@ -89227,21 +89227,67 @@ HybridModel:Zone,
        \type object-list
        \object-list ZoneNames
   A3 , \field Calculate Zone Internal Thermal Mass
-       \note Use measured temperature data to calculate zone temperature capacity multiplier
-       \note Zone Air Infiltration Rate and Zone Internal Thermal Mass calculations cannot be performed simultaneously
+       \note Use measured data (temperature, humidity ratio, or CO2 concentration) to calculate zone temperature capacity multiplier.
+       \note At leat one measured parameter should be provided to calculate the thermal mass.
        \type choice
        \key No
        \key Yes
        \default No
   A4 , \field Calculate Zone Air Infiltration Rate
-       \note Use measured temperature data to calculate zone air infiltration air flow rate
-       \note Zone Air Infiltration Rate and Zone Internal Thermal Mass calculations cannot be performed simultaneously
+       \note Use measured temperature data (temperature, humidity ratio, or CO2 concentration) to calculate zone air infiltration air flow rate.
+       \note At leat one measured parameter should be provided to calculate the thermal mass.
```

Comment:
add a note that A3 and A4 can not be YES at the same time (?), and other logics where apply.

Reply:
Only one of field A3 - Calculate Zone Internal Thermal Mass, A4 - Calculate Zone Air Infiltration Rate, and A5 - Calculate Zone People Count can be set to YES at a time. If A4  is defined and set to YES, one of the following fields (combinations) should be provided:
1. Measurements were conducted when HVAC is free-floating:
1.1 A6 - Zone Measured Air Temperature Schedule Name
1.2 A7 - Zone Measured Air Humidity Ratio Schedule Name
1.3 A8 - Zone Measured Air CO2 Concentration Schedule Name
2. Measurements were conducted when HVAC is on:
2.1 A6 - Zone Measured Air Temperature Schedule Name, A13 - Zone Input Supply Air Temperature Schedule Name, and A14 - Zone Input Supply Air Mass Flow Rate Schedule Name
2.2 A6 - Zone Measured Air Humidity Ratio Schedule Name, A14 - Zone Input Supply Air Mass Flow Rate Schedule Name, and A15 - Zone Input Supply Air Humidity Ratio Schedule Name
2.3 A8 - Zone Measured Air CO2 Concentration Schedule Name, A14 - Zone Input Supply Air Mass Flow Rate Schedule Name, and A16 - Zone Input Supply Air CO2 Concentration Schedule Name